packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -167,8 +167,6 @@ def _raise_if_file_not_on_disk(path: Path) -> None:
 
             if isinstance(converted_value, Path):
                 if converted_value.is_symlink():
-                    if not self._node_ports.follow_symlinks:
-                        raise SymlinksUsageIsDisabledError()
                     symlink_target_path = Path(os.readlink(converted_value))
```

Comment:
I think there was a misunderstanding here. My point was not that if follow_symlinks was true, to raise an exception ... there is no point in adding a variable if raises an exception if used. 

Aside of the name of the variable, the problem is that I see three feasible situations: 
1) stores the symlink reference and then restores the symlink back. (probably if the refereed data does not exist upon restoring then it should error)
2) stores the symlink referee (actual data) and restores the data. In this case, it ensures the data will be in place but will not be in sync anymore with the original.
3) some hybrid that stores both 1 and 2 and restores the data (if the original does not exists) or the symlink (if the original exists). 

I think you are implementing 2 but i can imagine some cases with 1.

Reply:
I think this is a bit of an overstretch. Our use case is a simple one.

When taking about this feature, I think of nodeports, as follows: node A can upload to its output a file which can be used by whatever type of node that accepts a file as inputs. With this in mind:
1. If you want to support this then you should only store a symlink which points to file that also gets uploaded. If this is the case I would encourage the user to use directly the file name or keep this symlink somewhere else. **I would not support this case**.
2. When node A uploads a symlink it uploads the content of that symlink. Node B will download the contents as if it would be the original file. Node B does not care about the origin of the data, it knows that what it has is the latest version. **Yes this is the current use case**.
3. I did not understand it and perhaps is already too complicated.

Basically yes, I am implementing 2 but I do not want to support 1 because I see no added value since we can instruct the users to do something else.

Reply:
ok. agreed to implement only 2. I have a use case for 1 and we will follow up on it. thx